containerfiles/stable

```diff
@@ -1,4 +1,4 @@
-FROM registry.playtron.one/internal/playtronos:1.0.0.13
+FROM ghcr.io/playtron-os/playtron-os:1.0.0.13
```

Comment:
What do you think about pulling the image name into an argument that can be overridden in CI?

E.g.
```docker
ARG IMAGE=ghcr.io/playtron-os/playtron-os
FROM ${IMAGE}:1.0.0.13
```